src/nvim/testdir/Makefile

```diff
@@ -26,7 +25,6 @@ SCRIPTS_DEFAULT = \
 ifneq ($(OS),Windows_NT)
   SCRIPTS_DEFAULTS := $(SCRIPTS_DEFAULT) \
            test17.out             \
-           test49.out             \
```

Comment:
Was this intentional? As much as I'd _like_ to remove this, it wasn't removed in the vim patch, and is still present on Vim HEAD.

Reply:
Good catch.  Fixed, and sorted the test files.